backend/internal/controllers/middleware/requireAuth.go

```diff
@@ -0,0 +1,41 @@
+package middleware
+
+import (
+	"context"
+	"net/http"
+	"strings"
+
+	"github.com/TritonSE/words-alive/internal/auth"
+)
+
+// HTTP middleware setting a value on the request context
+func RequireAuth(authenticator auth.Authenticator) func(http.Handler) http.Handler {
+
+	// middleware returned on calling RequireAuth
```

Comment:
IMO this comment is unnecessary, we all know what a `return` statement does :) Was there something else you were trying to say here?